<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoListPlugin</class>
 <widget class="QDockWidget" name="TodoListPlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Todo List</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="layout_tasks">
      <item>
       <widget class="QListView" name="tasks_list"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="tasks_buttons_layout">
        <item>
         <widget class="QPushButton" name="button_add_task">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_remove_task">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>36</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="general_tab">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="layout_name">
          <item>
           <widget class="QLabel" name="label_name">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="entry_name"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layout_description">
          <item>
           <widget class="QLabel" name="label_description">
            <property name="text">
             <string>Description:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="description_text"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="button_complete_task">
          <property name="text">
           <string>Complete Task</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_save_tasks">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>36</height>
       </size>
      </property>
      <property name="text">
       <string>Save Tasks</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="9"/>
 <resources/>
 <connections/>
</ui>
